serverless/README.md

```diff
@@ -69,20 +71,22 @@ To further configure your plugin, use the following custom parameters:
-| `env`                   | When set, the macro adds an `env` tag to all Lambda functions with the provided value.                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `service`               | When set along with a `forwarderArn`, the macro adds an `service` tag to all Lambda functions with the provided value. When set along with the `extensionLayerVersion`, the macro adds a `DD_SERVICE` environment variable is added to all lambda functions with the provided value.                                                                                                                                                                                                                |
+| `env`                   | When set along with a `forwarderArn`, the macro adds an `env` tag to all Lambda functions with the provided value. When set along with the `extensionLayerVersion`, the macro adds a `DD_ENV` environment variable is added to all lambda functions with the provided value.                                                                                                                                                                                                                        |
+| `version`               | When set along with a `forwarderArn`, the macro adds `version` tag to all Lambda functions with the provided value. When set along with the `extensionLayerVersion`, the macro adds a `DD_VERSION` environment variable is added to all lambda functions with the provided value.                                                                                                                                                                                                                   |
+| `tags`                  | A comma separated list of key:value pairs as a single string. When set along with `forwarderArn`, the macro parses the string and sets each key:value pair as a tag to all Lambda functions. When set along with `extensionLayerVersion`, a `DD_TAGS` environment variable is added to all Lambda functions with the provided value.                                                                                                                                                                |
```

Comment:
```suggestion
| `tags`                  | A comma separated list of key:value pairs as a single string. When set along with `extensionLayerVersion`, a `DD_TAGS` environment variable is added on all Lambda functions with the provided value.  When set along with `forwarderArn`, the macro parses the string and sets each key:value pair as a tag on all Lambda functions. 
|
```

Reply:
so when the extensionLayerVersion is added the macro doesn't parse the tags string and seperate the comma seperated key value pairs? I.e. would `tags = "key1:val1,key2:val2"` be added as `DD_TAGS = key1:val1,key2:val2`?

Reply:
Yeah with the extension DD_TAGS gets set directly as what was inputed as the `tags` I believe we later handle the parsing of DD_TAGS (probably in the extension but not sure because I haven't really touched this)